<launch>        
 
   <node pkg="ceres_control" type="ceres_control_app" name="ceres_control" output="screen">
	    <remap from="ceres_control/send_setpoint" to="mavros/setpoint/local_position"/>
	</node>
</launch>
